### PR TITLE
 📝 add a new doc to explain how RHMAP and digger work together

### PR DESCRIPTION
**Pull request by wei-lee**

### CODE REVIEW

**Comment by wei-lee** [comment]
@PhilipGough @odra @aliok can you guys review this doc?

@finp this is the doc page I was talking to you about.

File: docs/digger/digger-and-rhmap.asciidoc [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+=== Pre-build
+
+This stage includes all the steps before the build is started on Jenkins. 
+
+During this stage, RHMAP will make sure all the required resources are gathered, and the corresponding Jenkins job is updated to reflect the currect status of the project on RHMAP.
+For all the keys and certificates that are required for the build, RHMAP will also make sure they are saved in the Jenkins credential store.
+
+=== Build
+
+This stage includes all the steps from the build is started on Jenkins, until the build is completed.
```
Comment (PhilipGough) [inline docs/digger/digger-and-rhmap.asciidoc]: from ---> from when

File: docs/digger/digger-and-rhmap.asciidoc [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+During this stage, the build is done by Jenkins and RHMAP will just poll the logs from Jenkins and present the logs to the client.
+
+=== Post-build
+
+This stage includes all the steps from the build is completed on Jenkins, until the request is fully completed.
```
Comment (PhilipGough) [inline docs/digger/digger-and-rhmap.asciidoc]: As above

**Comment by PhilipGough** [comment]
@wei-lee two small grammar comments. Looks good

**Pull request merged**